blogposts/announcing-sourcegraph-3.3.md

```diff
@@ -10,86 +10,75 @@ heroImage: /sourcegraph-mark.png
+<div style="padding-left: 2rem">
 
-We’ve completely overhauled how repositories are mirrored from your code host so that it is easier to configure and more reliable. This benefits everyone, but especially organizations with thousands (or tens of thousands) of repositories.
+[**🛠️ Config-based repository selection**](#config-based-repository-selection)<br />
+Define add/exclude lists in external services instead of enabling/disabling.
 
-Sentry joins our list of developer tool integrations, and Swift developers get better symbols definition and better code navigation.
+**[🧯 Jump to Sentry issue from your code](#new-sentry-integration-with-the-sentry-sourcegraph-extension)**<br/>
+Error and exception handling code, annotated with Sentry links.
 
-Read on for the details, and thanks to our customers and community for reporting issues and providing feedback. You’re helping to make each Sourcegraph release the best one yet!
+**[🔌 Bitbucket Server integration](#bitbucket-server-integration)**<br />
+Better PR code reviews on Bitbucket Server with hover tool-tips and "go to definition".
 
-## Release highlights
+**[✔️Grant permissions on your code host](#grant-permissions-on-your-code-host)**<br />
+When using code hosts other than GitHub.com.
 
-<div style="padding-left: 2rem">
+[**🧭 "Go to definition" using dependency graph**](#go-to-definition-using-dependency-graph)<br />
+`import` / `require` statements used to help find the right definition.
 
-[**🛠️ New! Config based repository selection**](#repo-config)<br />
-Repositories now chosen via add/exclude lists and query rules.
+[**🦅 Swift code navigation**](#swift-code-navigation)<br />
+Fly around Swift code on Sourcegraph.com or your code host.
 
-**[🔌 New Sentry integration](#sentry) and [Bitbucket Server fixes](#bitbucket-fixes)**<br />
-Jump to Sentry from Sourcegraph and other integrations improvements.
+[**🗄 Symbols for the current file**](#symbols-for-the-current-file)<br />
+The symbols sidebar is now scoped to the current file/directory you’re viewing.
 
-[**🧭 More precise code intelligence**](#code-intel)<br />
-Symbols scoped to the current file and improved Swift support.
 <!--
 TODO(ryan): Show once 3.3 is available on DigitalOcean
 [**🚢 Deployments secure by default with TLS pre-configured**](#deployments)<br />
 AWS and DigitalOcean deployments with TLS via self-signed certs.
 -->
-[**📣 Making Sourcegraph a part of your dev tools stack**](#grok-sourcegraph)<br />
-Tips for integrating Sourcegraph and code search into your team's workflow.
 
-[**🔎 Search improvements**](#search)<br />
-Auto-fixing search queries and new option to search in large files (> 20Mb).
+[**🔎 Handle common search queries containing (unintended) invalid regexp**](#handle-common-search-queries-containing-unintended-invalid-regexp)<br />
+Searching for function call sites e.g. `throw(` now work as expected.
```

Comment:
```suggestion
Searching for function call sites such as `open(` now work as expected.
```

I recommend `such as` instead of `e.g.` because `e.g.` usually is offset by commas, which would interfere with the example. And `open(` not `throw(` because the latter is usually not followed by parens, like `throw new Error('foo')` not `throw('foo')`.